hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1708,4 +1708,17 @@ ContainerTokenSecretManager getContainerTokenSecretManager() {
     return containerTokenMgr;
   }
 
+  @Override
+  public List<String> getScmRatisRoles() throws IOException {
+    return getScmHAManager().getRatisServer().getRatisRoles();
```

Comment:
Can we format as String in a readable way, instead of the current format using StringBuilder?

Reply:
Also, it better to show the hostname than the nodeId in the UI.

Reply:
I believe this is already returning hostnames. In docker hostnames are scm1,scm2.

Reply:
Port looks too general, can we specifically mention ratis port?
As SCM has multiple RPC endpoints.

Reply:
Done.